src/fitnesse/wiki/PageCrawler.java

```diff
@@ -4,30 +4,189 @@
 
 import fitnesse.components.TraversalListener;
 
-public interface PageCrawler {
-  WikiPage getPage(WikiPagePath path);
+public class PageCrawler {
```

Comment:
I think this should be left as an interface.

Reply:
The reason why I removed it is that there were the interface and its only implementing class directly next to each other in the same package. Plus, the implementing class did not represent a proper concept of its own (It was simply called PageCrawlerImpl). So I tried to sharpen the concept here.

Can you please explain why you would prefer an interface plus an ...Impl class? (I don't understand this in general, so please bear with my question.)